### PR TITLE
Update example section for keeping browser open

### PR DESCRIPTION
**Pull request by Omjoa001**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ x ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [ ] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Comment by titusfortner** [comment]
Nice, thanks! Would you like to also add the reference to this code on the browser/chrome.md page? Let us know if we can help.

**Comment by harsha509** [comment]
Hi @Omjoa001 ,

Hope you're doing great!

Did you got a chance to look into above comments ?

Thanks,
Sri

File: examples/python/tests/browsers/test_chrome.py [hunk 1]
```diff
@@ -7,3 +7,9 @@ def test_basic_options():
     driver = webdriver.Chrome(options=options)
 
     driver.quit()
+
+def test_keep_browser_open():
+    chrome_options = Options()
```
Comment (harsha509) [inline examples/python/tests/browsers/test_chrome.py:12]: chrome_options = ChromeOptions()

**Review by harsha509 — CHANGES_REQUESTED** [review]

**Comment by harsha509** [comment]
HI @Omjoa001,

Thank you for the PR!

As the PR went stale, I have updated the changes with commit e544b2bfebb1752688adbbb4c8c5c2087b815e55 and 603367ec5fbbf89fa58b2a9bf69b4b69a9178f8b !

Thanks,
Sri Harsha

**Pull request closed without merging**